eng/pipelines/templates/stages/archetype-js-release.yml

```diff
@@ -229,7 +229,7 @@ stages:
     - job: PublishPackages
       # Run Integration job only if SetDevVersion is set to true or ( SetDevVersion is empty and job is a scheduled CI run)
       # If SetDevVersion is set to false then we should skip integration job even for scheduled runs.
-      condition: and(eq(variables['SetDevVersion'], 'true'), eq(variables['System.TeamProject'], 'internal'))
```

Comment:
I guess the old approach only worked if you set the variable at queue time (I think I knew that at some point :(). It would be good if we can figure out a way to have this set at a higher level but I'm not sure, maybe we can set it as an output variable or something like that.

Reply:
I think we can, but I didn't want to block this issue with JS while I figured it out.